Vireo_VS/VireoCommandLine.vcxproj

```diff
@@ -96,7 +96,7 @@
     <UseDebugLibraries>false</UseDebugLibraries>
     <PlatformToolset>v140</PlatformToolset>
     <WholeProgramOptimization>true</WholeProgramOptimization>
-    <CharacterSet>Unicode</CharacterSet>
+    <CharacterSet>MultiByte</CharacterSet>
```

Comment:
I was trying to match Release configuration with Debug configuration (except for optimization, debug info, etc, of course). I am not sure whether there was a reason why Debug was "MultiByte" to begin with, @spathiwa ? We don't seem to be using any multi-byte / Unicode strings yet so it does not matter now, I think ! If we want to go with "Unicode" or just choose "Not set", we should fix both Debug and Release builds.